doc/changes/changes_4.3.1.md

```diff
@@ -0,0 +1,9 @@
+# SQL Statement Builder 4.3.1, released 2020-XX-XX
+
+Code Name: Refactoring
+
+## Refactoring
+ 
+* #98: Refactored comparison and like class structure
+   The refactoring changed the internal representation of the `Comparison`.
+   The public API from `BooleanTerm` did however not change.
```

Comment:
I don't know I we should leave it like that or make a major release, since the API chaned. It depends on what we considder as public API.

Reply:
No, it does not.